reference/5.1/Microsoft.PowerShell.Core/About/about_Continue.md

```diff
@@ -53,12 +56,78 @@ for ($i = 0; $i -lt 10; $i++)
     {
         continue
         # Will not result in an infinite loop.
-        $i--;
+        $i--
     }
 }
 ```
 
-## SEE ALSO
+### Using a labeled continue in a loop
+
+A labeled `continue` statement terminates execution of the iteration and
+transfers control to the targeted enclosing iteration or switch statement
+label.
+
+In the following example, the innermost `for` is terminated when `$condition`
+is **True** and iteration continues with the second `for` loop at `labelB`.
+
+```powershell
+:labelA for ($i = 1; $i -le 10; $i++) {
+    :labelB for ($j = 1; $j -le 10; $j++) {
+        :labelC for ($k = 1; $k -le 10; $k++) {
+            if ($condition) {
+                continue labelB
+            } else {
+                $condition = Update-Condition
+            }
+        }
+    }
+}
+```
+
+## Using continue in a switch statement
+
+An unlabeled `continue` statement within a switch terminates execution of the
+current switch iteration and transfers control to the top of the switch to get
+the next input item.
+
+When there is a single input item `continue` exits the entire switch statement.
+When the switch input is a collection, the switch tests each element of the
+collection. The continue exits the current iteration and the switch continues
+with the next element.
```

Comment:
Several 'switch' and 'continue' elements need codefencing in these paragraphs.

Reply:
Fixed